lib/portlayer/storage/image_cache.go

```diff
@@ -88,7 +88,7 @@ func (c *NameLookupCache) GetImageStore(ctx context.Context, storeName string) (
 		// Add Scratch
 		scratch, err := c.DataStore.GetImage(ctx, store, Scratch.ID)
 		if err != nil {
-			log.Errorf("Error looking up scratch on %s: %s", store.String(), err)
+			log.Errorf("ImageCache Error: looking up scratch on %s: %s", store.String(), err)
```

Comment:
Done.  i still want a separate log line with the error received since that includes the vsphere error.